src/pages/PostCompose.jsx

```
@@ -220,6 +228,14 @@ export default function PostCompose() {
           brandName = u.brandName || u.brand?.name;
         }
       } catch {}
+      // Fallback to brand context passed via URL (from Brand sub-tab)
+      try {
+        const sp = new URLSearchParams(location.search);
+        const qBrandId = sp.get('brandId');
+        const qBrand = sp.get('brand');
+        if (!brandId && qBrandId) brandId = qBrandId;
+        if (!brandName && qBrand) brandName = qBrand;
+      } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)